src/analysis/complex/exponential.lean

```diff
@@ -1418,6 +1418,9 @@ lemma cpow_def (x y : ℂ) : x ^ y =
 
 @[simp] lemma cpow_zero (x : ℂ) : x ^ (0 : ℂ) = 1 := by simp [cpow_def]
 
+@[simp] lemma cpow_eq_zero_iff (x y : ℂ) : x ^ y = 0 ↔ x = 0 ∧ y ≠ 0 :=
+by { simp [cpow_def], split_ifs; simp [*, exp_ne_zero] }
```

Comment:
Non-terminating `simp`. Does it apply anything but `cpow_def`? If not, please use `simp only`.

Reply:
Done